.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Projections CI
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
Could use a matrix here to test various java versions (and maybe OSs)

Reply:
Added.